### ISSUE TITLE
GOPATH may contain multiple directories

### ISSUE THREAD

**Issue by smthpickboy** [drichardson/gogetprivate#2]
When GOPATH contains multiple directories, go get will use the first directory. I think gogetprivate should deal with this condition to mimic go get's behavior.

Thanks!

**Comment by drichardson** [comment]
Good idea. If you have a patch I can merge it right away, otherwise might be a bit before I can get to this.
